here-naksha-storage-http/src/main/java/com/here/naksha/storage/http/HttpStorage.java

```diff
@@ -51,31 +59,117 @@ public HttpStorage(@NotNull Storage storage) {
             properties.getSocketTimeout()));
   }
 
-  @Override
   public @NotNull IReadSession newReadSession(@Nullable NakshaContext context, boolean useMaster) {
     return new HttpStorageReadSession(context, useMaster, requestSender);
```

Comment:
Since this method is no longer used in Handlers or Tasks it can be removed completely